packages/checkup-plugin-ember-octane/src/results/octane-migration-status-task-result.ts

```diff
@@ -58,73 +47,63 @@ export default class OctaneMigrationStatusTaskResult extends BaseTaskResult impl
   }
 
   json() {
-    let nativeClassMigrationInfo = transformESLintReport(
-      ESLINT_MIGRATION_TASK_CONFIGS[ESLintMigrationType.NativeClasses],
-      this.esLintReport
-    );
-
-    let taglessComponentMigrationInfo = transformESLintReport(
-      ESLINT_MIGRATION_TASK_CONFIGS[ESLintMigrationType.TaglessComponents],
-      this.esLintReport
-    );
+    return {
+      meta: this.meta,
+      result: {
+        totalViolations: this.esLintReport.errorCount + this.templateLintReport.errorCount,
+        migrationTaskResults: this.migrationResults,
+      },
+    };
+  }
 
-    let glimmerComponentsMigrationinfo = transformESLintReport(
-      ESLINT_MIGRATION_TASK_CONFIGS[ESLintMigrationType.GlimmerComponents],
-      this.esLintReport
-    );
+  pdf() {
+    return this.migrationResults
+      .map((migrationResult) => this._createPieChartData(migrationResult))
+      .filter((item) => item !== undefined) as PieChartData[];
```

Comment:
I think you can use this:

```suggestion
      .filter(Boolean) as PieChartData[];
```

Reply:
adding